Data/Binary/Serialise/CBOR/Write.hs

```diff
@@ -524,7 +525,57 @@ doubleMP = withConstHeader 0xfb P.doubleBE
 breakMP :: P.BoundedPrim ()
 breakMP = constHeader 0xff
 
---TODO: optimised implementation for GMP
+#if defined(OPTIMIZE_GMP) && MIN_VERSION_integer_gmp(1,0,0)
```

Comment:
`MIN_VERSION_integer_gmp(1,0,0)` is implied by `OPTIMIZE_GMP`, see https://github.com/well-typed/binary-serialise-cbor/blob/master/cbits/cbor.h#L35